app.js

```diff
@@ -21,62 +21,18 @@ let draws = 0;
 
 
 
-startButton.addEventListener('click', {
-
-    const randomThrow = goComputer()
-    const checkeRadioBdutton = document.querySelector(':checked');
-    const userPick = checkedRadioButton.value
-    
-    
+startButton.addEventListener('click', () => { 
 
+    const currentComputerPick = randomThrow();
+    const checkedRadioButton = document.querySelector(':checked');
+    const currentUserPick = checkedRadioButton.value;
+ 
+    // ------------------------------------------------------------------------------------
+    // set event listeners to update state and DOM
+    // If picks are the same they tie. 
+ 
+    doesUserWin(currentUserPick, currentComputerPick)
```

Comment:
I made some notes about your 'select-rock', 'select-paper', and 'select-scissors' in your HTML, I'd recommend changing those to 'rock', 'paper', and 'scissors' there so your function will work. Beyond that just write some logic for how to add win, loss, and draw totals. You're close!